packages/components/src/components/common/loading-message/loading-message-styles.ts

```diff
@@ -0,0 +1,8 @@
+import { type Styles } from 'jss';
+import { getHiddenTextJssStyle } from '../../../styles';
+
+export const getFunctionalComponentLoadingMessageStyles = (): Styles<'status'> => {
+  return {
+    status: getHiddenTextJssStyle(),
```

Comment:
I think the style class should be named like the functional component is named to be in sync with other functional components.